app/src/main/java/org/hackillinois/android/view/home/HomeFragment.kt

```diff
@@ -81,6 +82,14 @@ class HomeFragment : Fragment(), CountdownManager.CountDownListener, EventsListA
     }
 
     private fun updateEventsList(events: List<Event>?) {
+        if (events?.isEmpty() == true) {
```

Comment:
Drop the "== true" or use ternary operators

Reply:
Only reason I did this is because it may return null, so adding the "== true" makes it work since null will not evaluate to false in kotlin. I could do something like `events?.isEmpty() ?: false`, though.